worldwide/natural-labels.json

```diff
@@ -64,17 +37,7 @@
       "properties": {
         "place_name": "Southern Appalacian Mountain Range, North Carolina",
         "zoom": 10,
-        "description": "A field of mountain-peak labels near Asheville, NC",
-        "highlights": [
-          {
-            "geometry_type": "Point",
-            "data_layer": "natural_label",
-            "data_layer_fields": {
-              "class": "landform",
-              "maki": "mountain"
-            }
-          }
-        ]
```

Comment:
I'm wary of us dropping highlight information altogether. I could definitely see us retaining some of these highlights as tags. This is a job that might require some manual curation, but I'd say it might be worth it. Curious to hear what you've been thinking!

Reply:
Dunno. This just seems like we are hard coding in some relationship to a tileset. The context this data provides can be fetched using tileQuery from the consumer if necessary?

Reply:
Ehh but tileQuery could return a good number of results, with no indication of what's actually important to look at? My take on tags is that the tileset piece is less important than creating some link between entries that might be useful to look at together— so for this particular feature, I'd be happy seeing `tags: ['mountain']` over `tags: ['landform-mountain']`. I think it's fair to say this is less important with the smaller gazetteers though, so I'm ok with dropping.

Either way, some of the features did contain tags that reflect the data (seeing `landform-mountain-peak` on the feature below this) so if we decide to drop, would be good to clean that up!